util/chunk/row_container.go

```diff
@@ -226,7 +228,15 @@ func (a *SpillDiskAction) Action(t *memory.Tracker) {
 	}
 	a.once.Do(func() {
 		atomic.StoreUint32(&a.c.exceeded, 1)
-		logutil.BgLogger().Info("memory exceeds quota, spill to disk now.", zap.String("memory", t.String()))
+		buffer := bytes.NewBufferString("\n")
+		fmt.Fprintf(buffer, "%s\"%s\"{\n", "", t.Label())
+		bytesLimit := t.GetBytesLimit()
+		if bytesLimit > 0 {
+			fmt.Fprintf(buffer, "%s  \"quota\": %s\n", "", t.BytesToString(bytesLimit))
+		}
+		fmt.Fprintf(buffer, "%s  \"consumed\": %s\n", "", t.BytesToString(t.BytesConsumed()))
+		buffer.WriteString("}\n")
+		logutil.BgLogger().Info("memory exceeds quota, spill to disk now.", zap.String("memory", buffer.String()))
```

Comment:
```suggestion
		logutil.BgLogger().Info("memory exceeds quota, spill to disk now.", zap.Int64("qonsumed", t.BytesConsumed()), zap.Int64("quota", t.GetBytesLimit()))
```

We can just print the plain text instead of JSON.

Reply:
s/qonsumed/consumed/?

Reply:
sorry about that.
@yzwqf Please fix this typo.